package.json

```diff
@@ -39,12 +24,9 @@
   "files": [
     "browser.mjs",
     "browser.d.ts",
-    "index.cjs",
     "helpers/*.js",
     "helpers/*",
     "index.mjs",
-    "yargs",
```

Comment:
Glad to see this file with no extension going away. 👍  (I suspect it may have caused a couple of the rare weird issues reported with third party tools getting confused about whether cjs or esm.)